spec/features/bidder_interacts_with_auction_spec.rb

```diff
@@ -85,4 +113,83 @@
     expect(page).to have_content("No bids yet.")
   end
 
+  scenario "Viewing bid history for running auction" do
+    Timecop.scale(36000) do
+      create_running_auction
+    end
+    path = auction_bids_path(@auction.id)
+    visit path
+
+    # sort the bids so that newest is first
```

Comment:
I think I'm going to keep the comments. Otherwise, there's not an easy way to infer that a column is for "amount" (only that it is the 2nd column from the left). I will probably refactor the sorting, though.